images/atomic-registry-quickstart/registry-project-template.json

```diff
@@ -0,0 +1,130 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "registry-project-request",
+        "creationTimestamp": null
+    },
+    "objects": [
+        {
+            "kind": "Project",
```

Comment:
I think this can be reduced down to just the role bindings - you shouldn't need to specify namespace anywhere in here if you remove the group binding and instead state which roles you want.